src/whylogs/core/types/typeddataconverter.py

```diff
@@ -74,4 +74,10 @@ def get_type(typed_data):
             dtype = TYPES.INTEGRAL
         elif isinstance(typed_data, str):
             dtype = TYPES.STRING
+        if isinstance(typed_data, list) or isinstance(typed_data, np.ndarray) or isinstance(typed_data, pd.Series):
```

Comment:
Seems like duplicate of `_is_array_like()` ?

Reply:
It is the same condition, but we are going to soon replace/remove the typeddataconverter so I didn't home the method there and was trying for a targeted scope fix. We could pull the _is_array_like out of ColumnProfile but I think it is better to focus on the next version of this API over growing the type inspection logic.

Reply:
But I think you are right, its still better to move this to typeddataconverter than to duplicate the condition even, udpated.